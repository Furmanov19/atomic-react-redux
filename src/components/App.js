import React from 'react';
import { ThemeProvider } from 'styled-components';
import Atom from 'components/atoms/Atom';
import theme from './themes/default';

function App() {
  return (
    <ThemeProvider theme={theme}>
      <Atom>Hello world</Atom>
    </ThemeProvider>
  );
}

export default App;
