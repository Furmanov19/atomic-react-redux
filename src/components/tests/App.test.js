import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import App from 'components/App';

test('App should render correctly', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
