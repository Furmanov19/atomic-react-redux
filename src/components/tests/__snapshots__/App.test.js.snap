// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<ThemeProvider
  theme={
    Object {
      "fonts": Object {
        "pre": "Consolas, Liberation Mono, Menlo, Courier, monospace",
        "primary": "Helvetica Neue, Helvetica, Roboto, sans-serif",
        "quote": "Georgia, serif",
      },
      "palette": Object {
        "alert": Array [
          "#ffa000",
          "#ffc107",
          "#ffd761",
          "#ffecb3",
        ],
        "danger": Array [
          "#d32f2f",
          "#f44336",
          "#f8877f",
          "#ffcdd2",
        ],
        "grayscale": Array [
          "#212121",
          "#414141",
          "#616161",
          "#9e9e9e",
          "#bdbdbd",
          "#e0e0e0",
          "#eeeeee",
          "#ffffff",
        ],
        "primary": Array [
          "#1976d2",
          "#2196f3",
          "#71bcf7",
          "#c2e2fb",
        ],
        "secondary": Array [
          "#c2185b",
          "#e91e63",
          "#f06292",
          "#f8bbd0",
        ],
        "success": Array [
          "#388e3c",
          "#4caf50",
          "#7cc47f",
          "#c8e6c9",
        ],
        "white": Array [
          "#fff",
          "#fff",
          "#eee",
        ],
      },
      "reversePalette": Object {
        "alert": Array [
          "#ffecb3",
          "#ffd761",
          "#ffc107",
          "#ffa000",
        ],
        "danger": Array [
          "#ffcdd2",
          "#f8877f",
          "#f44336",
          "#d32f2f",
        ],
        "grayscale": Array [
          "#ffffff",
          "#eeeeee",
          "#e0e0e0",
          "#bdbdbd",
          "#9e9e9e",
          "#616161",
          "#414141",
          "#212121",
        ],
        "primary": Array [
          "#c2e2fb",
          "#71bcf7",
          "#2196f3",
          "#1976d2",
        ],
        "secondary": Array [
          "#f8bbd0",
          "#f06292",
          "#e91e63",
          "#c2185b",
        ],
        "success": Array [
          "#c8e6c9",
          "#7cc47f",
          "#4caf50",
          "#388e3c",
        ],
        "white": Array [
          "#eee",
          "#fff",
          "#fff",
        ],
      },
      "sizes": Object {
        "maxWidth": "1100px",
      },
    }
  }
>
  <styled.span
    palette="grayscale"
  >
    Hello world
  </styled.span>
</ThemeProvider>
`;
